### PR TITLE
[doc] Rework "latex" document

### PR DESCRIPTION
**Pull request by stephenfin**
Remove some unnecessary indentation, consistently document option defaults, and fix some wrapping and whitespace.

### CODE REVIEW

**Comment by mgeier** [comment]
The diff of this PR is much easier to read when ignoring whitespace changes: https://github.com/sphinx-doc/sphinx/pull/7752/files?w=1

File: doc/latex.rst [hunk 1]
```diff
@@ -579,55 +612,68 @@ macros may be significant.
...
+    :Default: ``true``
 
     .. versionadded:: 1.5.2
        set this option value to ``false`` to recover former behaviour.
 
 ``inlineliteralwraps``
-    default ``true``. Allows linebreaks inside inline literals: but extra
-    potential break-points (additionally to those allowed by LaTeX at spaces
-    or for hyphenation) are currently inserted only after the characters
+    Boolean to specify is linebreaks are allowed inside inline literals: but
```
Comment (tk0miya) [inline doc/latex.rst]: typo?: is

**Review by tk0miya — COMMENTED** [review]
LGTM with nits. BTW, is there any reason to specify the master branch as a target? It would be much better to merge this into 3.x branch for update it easily.

File: doc/latex.rst [hunk 2]
```diff
@@ -65,433 +49,487 @@ avoid interpretation as escape sequences, or use raw strings (as is done here).
...
+   .. _`polyglossia`: https://ctan.org/pkg/polyglossia
+   .. _`babel`: https://ctan.org/pkg/babel
+
+   .. hint::
+
+      After modifiying a core LaTeX key like this one, clean up the LaTeX
+      build repertory before next PDF build, else left-over auxiliary
+      files are likely to break the build.
+
+    :Default:  ``'\\usepackage{babel}'`` (``''`` for Japenese documents)
```
Comment (mgeier) [inline doc/latex.rst]: There is too much indentation here, the "Default" ends up in the "hint" box and the hint itself becomes a blockquote.

File: doc/latex.rst (hunk 1 above)
Comment (stephenfin) [reply doc/latex.rst]: Whoops. Thanks

File: doc/latex.rst (hunk 2 above)
Comment (stephenfin) [reply doc/latex.rst]: Good catch. Done

**Comment by stephenfin** [comment]
> LGTM with nits. BTW, is there any reason to specify the master branch as a target? It would be much better to merge this into 3.x branch for update it easily.

Rebased onto `3.x`

File: doc/latex.rst [hunk 3]
```diff
@@ -65,433 +49,487 @@ avoid interpretation as escape sequences, or use raw strings (as is done here).
...
+   .. _`polyglossia`: https://ctan.org/pkg/polyglossia
+   .. _`babel`: https://ctan.org/pkg/babel
+
+   .. hint::
+
+      After modifiying a core LaTeX key like this one, clean up the LaTeX
+      build repertory before next PDF build, else left-over auxiliary
+      files are likely to break the build.
+
+   :Default:  ``'\\usepackage{babel}'`` (``''`` for Japenese documents)
```
Comment (mgeier) [inline doc/latex.rst]: Japanese

**Comment by mgeier** [comment]
I like these changes, except for the use of field lists for the "Default" values.

I think a single-entry field list looks strange.

What about just using a plain new paragraph?

```rst
Default: ...
```

**Comment by stephenfin** [comment]
> I like these changes, except for the use of field lists for the "Default" values.
> 
> I think a single-entry field list looks strange.
> 
> What about just using a plain new paragraph?
> 
> ```
> Default: ...
> ```

Works for me. Done.

**Comment by tk0miya** [comment]
Merged!

**Pull request merged**